staker/block_validator.go

```diff
@@ -496,7 +498,18 @@ func (v *BlockValidator) createNextValidationEntry(ctx context.Context) (bool, e
 	} else {
 		return false, fmt.Errorf("illegal batch msg count %d pos %d batch %d", v.nextCreateBatchMsgCount, pos, endGS.Batch)
 	}
-	entry, err := newValidationEntry(pos, v.nextCreateStartGS, endGS, msg, v.nextCreateBatch, v.nextCreatePrevDelayed)
+	var commitment *espressoTypes.Commitment
+	if v.config().Espresso {
+		_, jst, err := arbos.ParseEspressoMsg(msg.Message)
```

Comment:
This will return an error if `msg.Header.Kind != arbostypes.L1MessageType_L2Message`.